rasa_core/policies/ensemble.py

```diff
@@ -208,9 +208,29 @@ def probabilities_using_best_policy(self, tracker, domain):
                 max_confidence = confidence
                 result = probabilities
                 best_policy_name = 'policy_{}_{}'.format(i, type(p).__name__)
+
+        policy_names = [type(p).__name__ for p in self.policies]
+
+        # Trigger the fallback policy when ActionListen is predicted after
+        # a user utterance. This is done on the condition that: a fallback
+        # policy is present, there was just a user message and the predicted
+        # action is action_listen by a policy other than the MemoizationPolicy
+
+        if "FallbackPolicy" in policy_names:
```

Comment:
this breaks if we ever rename this, let's make it fail in that case:
`FallbackPolicy.__name__ in policy_names`